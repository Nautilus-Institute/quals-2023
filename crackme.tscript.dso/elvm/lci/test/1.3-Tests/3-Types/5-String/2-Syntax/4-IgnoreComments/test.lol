HAI 1.3
	VISIBLE "Lorem ipsum BTW dolor sit"
	VISIBLE "Lorem ipsum OBTW dolor TLDR sit"
KTHXBYE
