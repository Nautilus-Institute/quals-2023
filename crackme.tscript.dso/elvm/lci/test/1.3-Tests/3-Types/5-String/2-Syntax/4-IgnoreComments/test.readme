This test checks that strings which contain embedded comment keywords work
correctly.
