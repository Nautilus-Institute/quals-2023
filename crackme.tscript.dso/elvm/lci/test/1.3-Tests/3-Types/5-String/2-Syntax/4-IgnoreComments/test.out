Lorem ipsum BTW dolor sit
Lorem ipsum OBTW dolor TLDR sit
